bindata/network/multus/multus.yaml

```diff
@@ -455,3 +455,39 @@ spec:
           configMap:
             name: cni-binary-copy-script
             defaultMode: 0744
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: ip-reconciler
+  namespace: openshift-multus
+  labels:
+    tier: node
+    app: whereabouts
+spec:
+  # reconcile loop every 5 minutes, starting at the top of the hour
+  schedule: "*/5 * * * *"
```

Comment:
Oh, huh, I was thinking it meant "run 5 minutes after the hour", not "run every 5 minutes".

In the long term we should think about whether it makes more sense to just have some existing long-running pod handle this, rather than spinning up a pod every 5 minutes... (Maybe CNO could do it?)

Reply:
Turning whereabouts into a thick plugin (long running pod) and migrate the ip cleanup reconcile loop there is one of our goals. I can't speak for **when** though... But it's on our radar.